### PR TITLE
Add hook and command to list groups

### PR DESCRIPTION
**Pull request by tulir**

### CODE REVIEW

File: src/botprovisioner.ts [hunk 1]
```diff
@@ -471,6 +471,44 @@ Usage: \`listusers\``,
...
+				let reply = "";
+				for (const d of descs) {
+					const groups = await this.bridge.hooks.listGroups(d.puppetId);
+					reply += `### ${d.puppetId}: ${d.desc}:\n\n`;
+					for (const g of groups) {
+						const mxid = await this.bridge.groupSync.getMxid({
+							puppetId: d.puppetId,
+							groupId: g.id!,
+						});
+						const replyPart = ` - ${g.name}: [${g.name}](https://matrix.to/#/${mxid})\n`;
```
Comment (Sorunome) [inline src/botprovisioner.ts:497]:
```suggestion
						const replyPart = ` - ${g.name}: [${g.name}](https://matrix.to/#/+{mxid})\n`;
```

shouldn't it be this?

Comment (tulir) [reply src/botprovisioner.ts:497]: That's a javascript template string variable, not a Matrix identifier sigil

Comment (Sorunome) [reply src/botprovisioner.ts:497]: Oh woops, you're right

**Comment by Sorunome** [comment]
Thanks for the PR!

**Pull request merged**